opencti-platform/opencti-graphql/src/generated/graphql.ts

```diff
@@ -19921,6 +19921,8 @@ export type StixDomainObjectsFiltering = {
 
 export enum StixDomainObjectsOrdering {
   Score = '_score',
+  AttributeCount = 'attribute_count',
```

Comment:
Why do we have these additions?

Reply:
it is taken into account in the export if the list is sorted by those columns. The export is not possible (raise error) if the column is not in StixDomainObjectsOrdering.